plugin-offline/README.md

```diff
@@ -35,7 +35,7 @@ repositories {
 
 // In the app build.gradle file
 dependencies {
-	implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-offline:0.3.0-SNAPSHOT'
+	implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-offline:0.5.0-SNAPSHOT'
```

Comment:
This is also fixed in https://github.com/mapbox/mapbox-plugins-android/pull/810.